### PR TITLE
created HELP_WANTED.md(having list of other sources)(#57)

### LINKED ISSUE #57: Search more sources for books
Search more books sources from where they can be added to the Internet archive. at this time, supported : punjab digital library and google books.
The list of prospective sources could be added as 'Help Wanted' so that developers could develop bub to support the new sources for books too.

### PR DESCRIPTION
**Pull request by dvitiarora**
Added list of sources(along with available APIs) other than Google Books and Punjab Digital Library.
Fixes #57

### CODE REVIEW

File: HELP_WANTED.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Following are the list of other sources that can be used to add books to Internet Archive, along with a sample book link and web APIs available on respective websites:
+
+1.Project Gutenberg(https://www.gutenberg.org/)
```
Comment (coderwassananmol) [inline HELP_WANTED.md]: Source approved! :)

File: HELP_WANTED.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Following are the list of other sources that can be used to add books to Internet Archive, along with a sample book link and web APIs available on respective websites:
+
+1.Project Gutenberg(https://www.gutenberg.org/)	
+	-**Sample Link**: http://www.gutenberg.org/files/155/155-h/155-h.htm
+	-Web **API** for Project Gutenberg ebook metadata https://gutendex.com(For more info: https://github.com/garethbjohnson/gutendex)
+
+2.Open Book Publishers(https://www.openbookpublishers.com/)
```
Comment (coderwassananmol) [inline HELP_WANTED.md]: Source approved! :)

File: HELP_WANTED.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+1.Project Gutenberg(https://www.gutenberg.org/)	
+	-**Sample Link**: http://www.gutenberg.org/files/155/155-h/155-h.htm
+	-Web **API** for Project Gutenberg ebook metadata https://gutendex.com(For more info: https://github.com/garethbjohnson/gutendex)
+
+2.Open Book Publishers(https://www.openbookpublishers.com/)		
+	-**Sample Link**: https://www.openbookpublishers.com/product/106#0
+	-**API**: https://github.com/OpenBookPublishers/obp_institution_api
+
+3.Bookrix(https://www.bookrix.com/)
```
Comment (coderwassananmol) [inline HELP_WANTED.md]: The books are not for public domain. Please delete this from the file.

File: HELP_WANTED.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+	-**Sample Link**: https://www.bookrix.com/book.html?bookID=bx.shakespeare_1212071424.7241220474#3240,540,94606
+
+4.Library of Congress  (http://www.read.gov/books/)
```
Comment (coderwassananmol) [inline HELP_WANTED.md]: Source approved! :)

File: HELP_WANTED.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+	-**Sample Link**: http://www.read.gov/books/pageturner/aesops_fables/#page/8/mode/2up
+	-**API**: The loc.gov JSON API provides structured data about Library of Congress collections.All URLs start with https://www.loc.gov/ and need to include fo=json as a parameter to get JSON.No API key or authentication is required.For more info: https://libraryofcongress.github.io/data-exploration/
+
+5.ManyBooks(https://manybooks.net/)
```
Comment (coderwassananmol) [inline HELP_WANTED.md]: Please delete this one.

File: HELP_WANTED.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+	-**Sample Link**: https://manybooks.net/book/139515/read#epubcfi(/6/2[titlepage]!/4/1:0)
+
+6.Bloomsbury Collections(https://www.bloomsburycollections.com/)
```
Comment (coderwassananmol) [inline HELP_WANTED.md]: Source approved! :)

File: HELP_WANTED.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+	-**Sample Link**: https://www.bloomsburycollections.com/book/government-communication-cases-and-challenges/#0
+
+7.Directory of Open Access Journals(https://doaj.org/)
```
Comment (coderwassananmol) [inline HELP_WANTED.md]: Source approved! :)

**Comment by dvitiarora** [comment]
Changes done. Please review.

**Comment by Parikshit-Hooda** [comment]
@dvitiarora check my review. shouldn't take much work to make the requested changes.

**Comment by dvitiarora** [comment]
I have done the changes. Please review and let me know if any improvement is needed to be done.

**Comment by Parikshit-Hooda** [comment]
@rkpattnaik780 please approve.

**Pull request merged**